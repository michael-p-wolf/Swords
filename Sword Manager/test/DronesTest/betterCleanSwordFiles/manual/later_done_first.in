3 3
-1, 10, 100, 100, 100, 2, 500, "3", "3", "3", "4"
-1, 7, 100, 100, 100, 2, 600, "3", "3", "3", "4"
-1, 5, 100, 100, 100, 2, 700, "3", "3", "3", "4"
3, 100, 2, 500, "4"
5, 100, 2, 600, "4"
6, 100, 2, 700, "4"
