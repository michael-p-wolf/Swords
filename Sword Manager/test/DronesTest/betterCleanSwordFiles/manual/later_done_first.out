11, 8, 100, 2, 700, "4"
12, 7, 100, 2, 600, "4"
13, 7, 100, 2, 500, "4"
